src/command/vanilla/TpCommand.ts

```diff
@@ -1,121 +1,115 @@
-import CommandParameter, { CommandParameterType } from "../../network/type/CommandParameter";
-import Player from "../../player";
-import Command from "../";
+import CommandParameter, { CommandParameterType } from '../../network/type/CommandParameter';
+import Player from '../../player';
```

Comment:
I'm not sure if we're using single or double quote. Consult @HerryYT and see what he thinks.